### DISCUSSION TITLE
Please show gitlab.com repo stats on npmjs.com package pages

### DISCUSSION THREAD

**Discussion by rhendric** [npm/feedback#422] [Website] [answered]
Previously:
npm/www#35
https://npm.community/t/393
npm/cli#488

I hope I'm not coming across as rude by continually re-posting this! This is something that's important to me and I don't want it to be lost as npm hops between various places to track feature requests.

To repeat myself: it would be great if packages hosted on gitlab.com got the same perks from npmjs.com as packages hosted on github.com—the perks I see right now are showing the number of open issues and pull requests, and linking to the appropriate pages for each.

I recognize that npm Inc. and GitHub are part of the same corporate family these days. Some people might cynically assume that because of this, of course npm Inc. wouldn't do anything to improve the experience of using a GitHub competitor. I hope they'd be wrong, but I guess we'll see.

**Comment by MylesBorins** [comment, marked as answer]
hey @rhendric thanks for the suggestion. At this time enhancing the package page with additional metadata from GitLab is not something that we would be able to prioritize. The majority of our engineering effort is going into improvements to npm 7, general platform stability, and features related to supply chain security. We are in the process of doing our quarterly planning right now and will be updating https://github.com/npm/roadmap/projects/1 accordingly.

We do have other backlogged features on the roadmap for improving the experience of understanding details about a package, for example surfacing more data about the format of a package. As we have more capacity to focus on general improvements to the package landing page our efforts are likely to be focused on initiatives such as surfacing more internal details about how a package is used as this type of information is platform independent.
